core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts

```diff
@@ -124,6 +124,7 @@ export class ConversationSkinComponent {
   isInPreviewMode: boolean;
   isIframed: boolean;
   hasFullyLoaded = false;
+  progressReminderIsShown = false;
```

Comment:
Could you explain the conditions for setting / unsetting this flag please? Asking since I don't see that we set it to false anywhere except on initialization.